client/src/pages/CreateSurvey/index.jsx

```diff
@@ -43,36 +41,30 @@ const CreateSurvey = ({ history }) => {
     modalStyle,
     isConfirming,
   } = useSelector((state) => state.createSurveyReducer);
-
   const { recipientIds } = useSelector((state) => state.employeeTableReducer);
   const dispatch = useDispatch();
-
   useEffect(() => {
     dispatch({ type: 'RESET_CREATE_SURVEY_MODAL_STATE' });
-    dispatch({ type: 'RESET_SURVEY_DATA' });
+    // dispatch({ type: 'RESET_SURVEY_DATA' });
```

Comment:
are these commented because of what João was saying? or another reason?

Reply:
yes! for now, i think this will change once we merge together

Reply:
actually no, this is on create survey so not mine!